src/main/java/com/statful/client/VertxMetricsImpl.java

```diff
@@ -3,25 +3,23 @@
 import com.statful.collector.*;
 import com.statful.sender.Sender;
 import com.statful.sender.SenderFactory;
+import io.vertx.core.Verticle;
 import io.vertx.core.Vertx;
-import io.vertx.core.eventbus.EventBus;
-import io.vertx.core.http.HttpClient;
+import io.vertx.core.datagram.DatagramSocketOptions;
 import io.vertx.core.http.HttpClientOptions;
-import io.vertx.core.http.HttpServer;
 import io.vertx.core.http.HttpServerOptions;
-import io.vertx.core.metrics.impl.DummyVertxMetrics;
+import io.vertx.core.net.NetClientOptions;
+import io.vertx.core.net.NetServerOptions;
 import io.vertx.core.net.SocketAddress;
-import io.vertx.core.spi.metrics.HttpClientMetrics;
-import io.vertx.core.spi.metrics.HttpServerMetrics;
-import io.vertx.core.spi.metrics.PoolMetrics;
+import io.vertx.core.spi.metrics.*;
 
 import java.util.concurrent.LinkedBlockingQueue;
 
 /**
  * VertxMetrics SPI implementation for statful metrics collection
  * Extending DummyVertxMetrics to avoid having to extend all methods even if we don't want to implement them
```

Comment:
Comment no longer applicable?

Reply:
The comment also doesn't make sense anymore because the methods it refers to are now default. I'll remove it

Reply:
Neat, it annoyed me like hell when I implemented it like this.